src/main/java/net/earthcomputer/clientcommands/command/arguments/ExpressionArgumentType.java

```diff
@@ -278,7 +278,7 @@ private static boolean isAllowedNumber(char c) {
 
     public static abstract class Expression {
         public String strVal;
-        public abstract double eval();
+        public abstract double eval() throws StackOverflowError;
```

Comment:
Subclasses of `java.lang.Error` don't need throws declarations.

Reply:
I know - I just wanted to be verbose. What about a `@throws` annotation instead?

Reply:
Ah, it's for documentation purposes, that's fine then. Maybe it should go on `getParsedTree` then too.